Gems/Atom/RHI/DX12/Code/Source/RHI/FrameGraphExecuteGroupMerged.cpp

```diff
@@ -29,11 +29,11 @@ namespace AZ
             m_workRequest.m_signalFence = m_scopes.back()->GetSignalFenceValue();
             m_workRequest.m_commandLists.resize(1);
 
-            AZStd::vector<RHI::ScopeId> scopeIds;
-            scopeIds.reserve(m_scopes.size());
+            AZStd::vector<InitMergedRequest::ScopeEntry> scopeEntries;
```

Comment:
you will need to update the metal version as well. It should be similar to dx12

Reply:
Updated.